src/utils/Colors.js

```diff
@@ -0,0 +1,13 @@
+export const Colors = {
```

Comment:
use all caps for the object name ie. COLORS

Reply:
Noted